doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -205,18 +205,15 @@ In Keystone v1 and v2 (both deprecated), the term "tenant" was used in OpenStack
 OpenStack offers multi-tenancy by means of resource (compute, network and storage)separation via projects. OpenStack offers ways to share virtual resources between projects while maintaining logical separation. As an example, traffic separation is provided by creating different VLAN ids for neutron networks of different projects. As another example, if host separation is needed, nova scheduler offers AggregateMultiTenancyIsolation scheduler filter to separate projects in host aggregates. Thus, if a host in an aggregate is configured for a particular project, only the instances from that project are placed on the host. Overall, tenant isolation ensures that the resources of a project are not affected by resources of another project.
 
 <a name="3.3.3"></a>
-### 3.3.3. Host aggregates providing resource pooling
-Availability zones: provide resiliency and fault tolerance for VM deployments, by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms
+### 3.3.3. Cloud partitioning: Host Aggregates, Availability Zones
+Cloud administrators can partitioned the hosts within an OpenStack cloud using 2 mechanisms: Host Aggregates and Availability Zones.
```

Comment:
@karinesevilla As you know AZs are host aggregates that are made visible to tenants and with the restriction that hosts can only belong to one AZ. (Applies also to later content). So, actually there is only onemechanism for partitioning.

Typo: "Cloud administrators can partitioned ..." ---> "Cloud administrators can partition ..."

Reply:
@goyal You are right, I modified the text accordingly with a second commit